nvflare/app_common/filters/svt_privacy.py

```diff
@@ -23,7 +23,7 @@
 
 class SVTPrivacy(Filter):
     def __init__(self, fraction=0.1, epsilon=0.1, noise_var=0.1, gamma=1e-5, tau=1e-6):
-        """implementation of the standard Sparse Vector Technique (SVT) differential privacy algorithm.
+        """Implementation of the standard Sparse Vector Technique (SVT) differential privacy algorithm.
```

Comment:
Do we have a reference paper for this "standard Sparse Vector Technique (SVT)"?

@holgerroth do you know?

Maybe we can add it here.